mslib/msui/editor.py

```diff
@@ -537,11 +686,8 @@ def remove_option_handler(self):
                 else:
                     non_removable.append(index)
 
-        # dialog_box = QtWidgets.QMessageBox.question(self, 'Remove option',
-        #                 'Are you sure about removing the selected options?',
-        #                 QtWidgets.QMessageBox.Yes, QtWidgets.QMessageBox.No)
-        # if dialog_box == QtWidgets.QMessageBox.No:
-        #     return
+        # self.error_dialog = QtWidgets.QErrorMessage()
```

Comment:
if the functionality is not needed, better remove it completly ;)

Reply:
I'm still thinking about how to include an confirmation message. I'll remove it.

Reply:
You could leave it but also add a `# ToDo` explaining what should be achieved. 

Later someone could have an idea how to solve it.  Without a ToDo likly someone on the next refactoring will remove it.

Reply:
Indeed, commenting something out is typically not a good idea - long term wise. As such it is always good to remove that before mergeing it.